dataactbroker/routeUtils.py

```diff
@@ -28,6 +28,8 @@ def run_instance_function(accountManager, accountFunction, getSystemEmail = Fals
                 else:
                     # Currently no functions with user but not credentials flag
                     raise ValueError("Invalid combination of flags to run_instance_function")
+            elif(addTrueFlag):
```

Comment:
I'm a bit torn on this. The `addTrueFlag` wasn't super clear to me, but I'm not in the part of the code base much. Can we page @nmonga91 for a consult?